test/queue_test.rb

```diff
@@ -4,19 +4,28 @@
 
 class ListTest < Minitest::Test
   def test_queue_count_in_event
```

Comment:
you could be more specific in this test name

```def test_queue_count_starts_at_zero```

Reply:
make your test names more [imaginary finger quotes] robust [imaginary finger quotes] 😸